cfgov/v1/models/base.py

```diff
@@ -131,6 +131,14 @@ class CFGOVPage(Page):
         ObjectList(settings_panels, heading='Configuration'),
     ])
 
+    def get_authors(self):
+        """ Returns a sorted list of authors. Default is alphabetical """
+        return self.alphabetize_authors()
+
+    def alphabetize_authors(self):
+        """ Alphabetize authors of this page by last name """ 
+        return sorted(self.authors.names(), key=lambda x: x.split(" ")[-1])
```

Comment:
another edge case where i can see this logic wouldn't hold up correctly is the case where there is no space.

```
'Duck'.split()[-1]
'k'
```

That would sort on 'k', not 'Duck', which i'm not sure is the intended behavior.